packages/itwinui-react/src/core/ExpandableBlock/ExpandableBlock.tsx

```diff
@@ -135,11 +136,7 @@ export const ExpandableBlock = (props: ExpandableBlockProps) => {
           <div className='iui-title'>{title}</div>
           {caption && <div className='iui-caption'>{caption}</div>}
         </span>
-        {icon &&
-          React.cloneElement(icon, {
-            className: cx('iui-status-icon', icon.props.className),
-            'data-iui-icon-color': status,
-          })}
+        {icon && <Icon fill={status}>{icon}</Icon>}
```

Comment:
I want to leave it unchanged to avoid any potential breaking changes for anyone who might be relying on those classes.

Reply:
Is updating the dom of `ExpandableBlock.tsx` a breaking change? E.g. if users have selectors with the old dom tree.

If so, can we update `ExpandableBlock.tsx` to use `<Icon/>` instead of `.iui-status-icon` in a minor release (according to the [changeset message](https://github.com/iTwin/iTwinUI/pull/854/files#diff-858caa1acb0daf69f63eb77bf057bf99303b567ab8f8d43d376f8838ddfeb503R2))?

Reply:
well, the itwinui-react API is unaffected. Users should not be targeting internal classes produced by our react components

whereas for itwinui-css, we are expected to keep supporting the old class names because someone might be consuming them directly.